main.go

```diff
@@ -40,6 +40,7 @@ Options:
     -Q --query      List all dependencies.
         -o          List only already-vendored dependencies.
     -C --clean      Detect all unused vendored dependencies and remove it.
+    -t --testing    Include testing dependencies.
```

Comment:
may be here we should write something like `Include dependencies from tests?` What you think about it?